back/engines/commercial/insights/app/controllers/insights/web_api/v1/stats_inputs_controller.rb

```diff
@@ -0,0 +1,27 @@
+module Insights
+  module WebApi::V1
+    class StatsInputsController < ::ApplicationController
+      def inputs_count
+        inputs = Insights::InputsFinder.new(view, counts_params, false).execute
```

Comment:
(pico: I would have probably opted for an option here to avoid the mysterious `false`, but I'm very picky here 😋)